source/mongodump.txt

```diff
@@ -321,6 +321,22 @@ If you're using :ref:`views <views-landing-page>`:
 - To export the documents in a view, use
   :option:`--viewsAsCollections <mongodump --viewsAsCollections>`.
 
+Metadata Output
+~~~~~~~~~~~~~~~
+
+If :binary:`~bin.mongodump` outputs to a directory, the output contains
+collection metadata which includes the following: 
+
+- Collection name
+- Collection type
+- Creation options
+- UUIDs
+- Indexes
+
+If :binary:`~bin.mongodump` outputs to standard output, the output does
```

Comment:
[optional suggestion] It might be helpful to specify that standard output refers to (`stdout`) similar to how it is in line 813!

```suggestion
If :binary:`~bin.mongodump` outputs to standard output (``stdout``), the output does
```

Reply:
Done!